projects/SMARDEX-P2P-Lending/index.js

```diff
@@ -19,49 +21,56 @@ const tokenMetricsQuery = `{
   }
 }`;
 
-const getTokenMetrics = async (subgraphUrl) => {
+const getTokenMetrics = async (subgraphUrl, chainName) => {
   const result = await fetch(subgraphUrl, {
     method: "POST",
     headers,
     body: JSON.stringify({
       query: tokenMetricsQuery,
     }),
   }).then((res) => res.json());
+  
   return result?.data?.tokenMetrics_collection || [];
 };
 
 
-async function getP2pData(subgraphUrl, isBorrowed = false) {
-  const tokenMetrics = await getTokenMetrics(subgraphUrl);
+async function getP2pData(subgraphUrl, isBorrowed = false, chainName = '') {
```

Comment:
You can set `chainName` as mandatory second argument so `isBorrowed` remain optional
```suggestion
async function getP2pData(subgraphUrl, chainName, isBorrowed = false) {
```